File: Assignment 01/Program 22/prog22.py
#Write a Python program to get the length of an array.

import array as arr

a1 = arr.array('i')
n = int(input("Enter the no. of elements: "))
for i in range(n):
    val = int(input("Enter the element: "))
    a1.append(val)

count = 0
for i in range(n):
    count+=1
print("The length of the array is :",count)

File: Assignment 01/Program 22/prog22.c
//Write a Python program to get the length of an array.

#include<stdio.h>
#include<stdlib.h>

int main(){
    int n;
    printf("Enter the number of inputs: ");
    scanf("%d",&n);
    int *arr = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        printf("Enter the number %d: ",i+1);
        scanf("%d",(arr+i));
    }
    int count = 0;
    for (int i = 0; i < n; i++)
    {
        count++;
    }
    printf("The length of the array is : %d",count);
    free(arr);
    return 0;
}